linera-service/Cargo.toml

```diff
@@ -19,6 +19,7 @@ wasmtime = ["linera-execution/wasmtime", "linera-storage/wasmtime"]
 rocksdb = ["linera-views/rocksdb", "linera-core/rocksdb", "linera-storage/rocksdb"]
 aws = ["linera-views/aws", "linera-core/aws", "linera-storage/aws"]
 scylladb = ["linera-views/scylladb", "linera-core/scylladb", "linera-storage/scylladb"]
+net_up_kubernetes = ["dep:k8s-openapi", "dep:kube", "dep:pathdiff"]
```

Comment:
`kubernetes` seems like a logical name. The goal is to control the inclusion of a set of libraries and features related to kubernetes.